javascript/src/index.ts

```diff
@@ -590,19 +590,25 @@ export interface WebhookUnbrandedRequiredHeaders {
   "webhook-signature": string;
 }
 
+export interface WebhookOptions {
+  raw?: boolean;
```

Comment:
```suggestion
  format?: "raw";
```
We may support other formats going forward.